storage_service/locations/api/urls.py

```diff
@@ -19,6 +19,18 @@
 urlpatterns = [
     url(r'', include(v1_api.urls)),
     url(r'v1/sword/$', views.service_document, name='sword_service_document'),
+    url(r'v1/doc/',
+        include('tastypie_swagger.urls', namespace='v1_api_swagger'),
+        kwargs={
+            'tastypie_api_module': v1_api,
+            'namespace': 'v1_api_swagger',
+            'version': '2.0'}),
     url(r'', include(v2_api.urls)),
     url(r'v2/sword/$', views.service_document, name='sword_service_document'),
+    url(r'v2/doc/',
+        include('tastypie_swagger.urls', namespace='v2_api_swagger'),
+        kwargs={
+            'tastypie_api_module': v2_api,
+            'namespace': 'v2_api_swagger',
+            'version': '1.0'}),
```

Comment:
@jraddaoui are the versions mixed up here? It looks like the v2/ endpoints are associated to version 1.0 and the v1/ endpoints are associated to version 2.0.

Reply:
Hi @jrwdunham,

Yes, correct namespace and module but swapped version string, sorry ;)